app/src/main/java/org/greenstand/android/TreeTracker/signup/CredentialEntryView.kt

```diff
@@ -1,20 +1,15 @@
 package org.greenstand.android.TreeTracker.signup
 
-import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.PaddingValues
-import androidx.compose.foundation.layout.Row
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.padding
-import androidx.compose.foundation.layout.size
```

Comment:
Make sure your auto formatting does not add '*'. Please undo these import changes.